app/test_main.py

```diff
@@ -1,3 +1,56 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestAnimalsYearsTransfer:
+    @pytest.mark.parametrize(
+        "cat_years,dog_years,expected_results",
+        [
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="should return [0, 0] if animals ages are 0's too."
+            ),
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="should return [0, 0] if animals ages are less then 15"
+            ),
+            pytest.param(
+                17,
+                17,
+                [1, 1],
+                id="should return [1, 1] if animals ages are >= 15 and < 23"
+            ),
+            pytest.param(
+                27,
+                27,
+                [2, 2],
+                id="should return [2, 2] if 24 <= `cat_years` < 28 "
+                   "and 24 <= `dog_years` < 29"
```

Comment:
The test case description 'should return [2, 2] if 24 <= `cat_years` < 28 and 24 <= `dog_years` < 29' seems inconsistent with the logic. The condition for `dog_years` should be '24 <= `dog_years` < 28' to match the expected result [2, 2].